### PR TITLE
Create `corrr` app

### PR DESCRIPTION
**Pull request by rich-iannone**
This is a Tableau extension that shows a correlation plot based on a single data table (numeric columns are automatically selected from a specified table).

### CODE REVIEW

**Comment by jcheng5** [comment]
I'm getting `Error: This tidyselect interface doesn't support predicates yet.` What version of tidyselect are you using?

**Comment by rich-iannone** [comment]
I'm using tidyselect 1.1.0.

**Comment by rich-iannone** [comment]
Also, don't merge any of this. Barret met up with me and made some improvements on my local branch that obviates the need for the promise pipe.

File: examples/corrr/app.R [hunk 1]
```diff
@@ -38,9 +37,10 @@ server <- function(input, output, session) {
   output$plot <- renderPlot({
 
     df() %...>% {
-      dplyr::select(., tidyselect:::where(is.numeric)) %>%
-        corrr::correlate(x = ., quiet = TRUE) %>%
-        corrr::rplot() +
+
+      numeric_cols <- names(which(vapply(., FUN.VALUE = logical(1), FUN = is.numeric)))
+
+      corrr::rplot(corrr::correlate(x = .[, numeric_cols], quiet = TRUE)) +
```
Comment (jcheng5) [inline examples/corrr/app.R]:
```suggestion
      corrr::rplot(corrr::correlate(x = .[, numeric_cols, drop = FALSE], quiet = TRUE)) +
```

File: examples/corrr/app.R [hunk 2]
```diff
@@ -38,9 +37,10 @@ server <- function(input, output, session) {
   output$plot <- renderPlot({
 
     df() %...>% {
-      dplyr::select(., tidyselect:::where(is.numeric)) %>%
-        corrr::correlate(x = ., quiet = TRUE) %>%
-        corrr::rplot() +
+
+      numeric_cols <- names(which(vapply(., FUN.VALUE = logical(1), FUN = is.numeric)))
```
Comment (jcheng5) [inline examples/corrr/app.R]: What happens if you leave of `names` and `which`?

Comment (rich-iannone) [reply examples/corrr/app.R]: Works well! Sorry I missed that.

**Pull request merged**